src/core/include/openvino/op/squeeze.hpp

```diff
@@ -32,8 +33,16 @@ class OPENVINO_API Squeeze : public Op {
 
     bool is_dynamic() const override;
 
+    bool get_pytorch_dynamic_rank() const {
+        return m_pytorch_dynamic_rank;
+    }
+    void set_pytorch_dynamic_rank(bool torch) {
+        m_pytorch_dynamic_rank = torch;
+    }
```

Comment:
Move implementation to cpp file.

It looks like setter is not required now can be added later when requried.

Reply:
done